src/ui/mainwindow.cc

```diff
@@ -1269,16 +1291,25 @@ void MainWindow::applyQtStyleSheet( QString const & addonStyle, QString const &
   }
 #endif
 
+#if !defined( Q_OS_WIN )
+  if ( interfaceStyle == "Default" ) {
+  #if defined( Q_OS_LINUX )
+    QApplication::setStyle( QStyleFactory::create( defaultInterfaceStyle ) );
+  #else
+    QApplication::setStyle( QStyleFactory::create( "Fusion" ) );
```

Comment:
can set `defaultInterfaceStyle="Fusion"` on macos.

then this 5 lines can be reduced to 
`QApplication::setStyle( QStyleFactory::create( defaultInterfaceStyle ) );`

Reply:
Yes, you are right.

New code is pushed.